containers/Dockerfile.buildenv

```diff
@@ -211,16 +210,6 @@ RUN ln -s $(which clang++-10) /usr/local/bin/clang++ \
 # building with Charm++ in the `tmp` directories.
 #
 # We do not build  with debug symbols and build with O2 to reduce build size.
-RUN git clone --single-branch --branch ${CHARM_GIT_TAG} --depth 1 \
```

Comment:
In the `RUN` command below, Charm 6.10.2 is installed without the space-saving measures you added here (cloning a single branch etc). Do you want to move those CLI args over?

Reply:
good idea :) I'll do that!

Reply:
I have no further requested changes